File: src/main/java/com/group/InternMap/Dto/RecruiterRegistrationDTO.java
package com.group.InternMap.Dto;

import com.group.InternMap.Model.User.Company.Company;
import com.group.InternMap.Model.User.Company.Recruiter;

public class RecruiterRegistrationDTO {
    private Recruiter user;
    private Company company;

    public Recruiter getUser() {
        return user;
    }

    public void setUser(Recruiter user) {
        this.user = user;
    }

    public Company getCompany() {
        return company;
    }

    public void setCompany(Company company) {
        this.company = company;
    }
}

File: Frontend/app/Model/DTO/RecruiterRegistrationDTO.ts
import type { Recruiter } from "../Users/Recruiter";
import { Company } from "../Company";

export class RecruiterRegistrationDTO {

    user: Recruiter;
    company: Company;

    constructor(user: Recruiter, company: Company) {
        this.user = user;
        this.company = company;
    }
}